tests/integration/api_v2/rpc/transactions.test.js

```diff
@@ -136,34 +137,29 @@ describe('Method get.transactions', () => {
 		});
 
 		it('invalid sender address -> empty response', async () => {
-			const response = await requestTransactions({ sender: 'lsydxc4ta5j43jp9ro3f8zqbxta9fn6jwzjucw7yj' });
-			expect(response).toMap(emptyResponseSchema);
-			const { result } = response;
-			expect(result).toMap(emptyResultEnvelopeSchema);
+			const response = await requestTransactions({ senderAddress: 'lsydxc4ta5j43jp9ro3f8zqbxta9fn6jwzjucw7yj' });
+			expect(response).toMap(invalidParamsSchema);
 		});
 	});
 
 	describe('is able to retrieve list of transactions using recipient attributes', () => {
 		it('known recipient address -> ok', async () => {
 			const response = await requestTransactions({
-				recipient: refTransaction.asset.recipientAddress,
+				recipientAddress: refTransaction.asset.recipientAddress,
 			});
 			expect(response).toMap(jsonRpcEnvelopeSchema);
 			const { result } = response;
 			expect(result.data).toBeInstanceOf(Array);
 			expect(result.data.length).toBeGreaterThanOrEqual(1);
 			expect(result.data.length).toBeLessThanOrEqual(10);
 			expect(response.result).toMap(resultEnvelopeSchema);
-			result.data.forEach(transaction => expect(transaction)
-				.toMap(transactionSchemaVersion5, { recipient: refTransaction.asset.recipientAddress }));
+			result.data.forEach(transaction => expect(transaction).toMap(transactionSchemaVersion5));
```

Comment:
Could you please add an assertion to validate that all the `recipientAddress` in all the transactions in the response matches the address in the request?

Please do the same in the test case corresponding to `senderAddress` and in the HTTP tests as well.

Reply:
Done!